lib/server.js

```diff
@@ -110,9 +110,9 @@ Server.prototype._onConnection = function(socket) {
         if (intercept) {
           socket.write(BUF_REP_INTR_SUCCESS);
           socket.removeListener('error', onErrorNoop);
-          process.nextTick(function() {
-            socket.resume();
-          });
+          //process.nextTick(function() {
+            //socket.resume();
+          //});
```

Comment:
Do you set up a listener `socket.on('data', ...)` right after calling `accept(true)`?

If not, you can pause the socket again with:

```javascript
socket = accept(true);
process.nextTick(function() {
  socket.pause();
});
```